ddp-workspace/projects/ddp-singular/src/app/components/activity/activity.component.ts

```diff
@@ -21,11 +21,6 @@ export class ActivityComponent extends ActivityRedesignedComponent {
     return this.model && this.model.activityCode === ActivityCode.Prequal;
   }
 
-  protected scrollToTop(): void {
-     document.body.firstElementChild.scrollTo(0,0);
-  }
```

Comment:
Removed the excess method.
The component uses same method in its parent **ActivityComponent** 
(`ddp-sdk\src\lib\components\activityForm\activity.component.ts`)